<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormQtMessage</class>
 <widget class="QWidget" name="FormQtMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Message</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,100,1">
   <item>
    <layout class="QHBoxLayout" name="hlayout" stretch="1,1,2">
     <item>
      <widget class="QCheckBox" name="chkbx_dbg">
       <property name="text">
        <string>Debug View</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkbx_info">
       <property name="text">
        <string>Info View</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vlayout">
     <item>
      <widget class="QTabWidget" name="tabwgt_dbg">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Debug View</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="1" column="2">
          <widget class="QToolButton" name="tbtn_clear_dbg">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="hspacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>216</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="tbtn_export_dbg">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QListWidget" name="listwgt_dbg"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabwgt_info">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Info View</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="2" column="1">
          <spacer name="hspacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>235</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="tbtn_clear_info">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="tbtn_export_info">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QListWidget" name="listwgt_info"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkbx_dbg</tabstop>
  <tabstop>chkbx_info</tabstop>
  <tabstop>tabwgt_dbg</tabstop>
  <tabstop>listwgt_dbg</tabstop>
  <tabstop>tbtn_export_dbg</tabstop>
  <tabstop>tbtn_clear_dbg</tabstop>
  <tabstop>tabwgt_info</tabstop>
  <tabstop>listwgt_info</tabstop>
  <tabstop>tbtn_export_info</tabstop>
  <tabstop>tbtn_clear_info</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
